recipes/r-cffr/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '0.1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cffr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/cffr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/cffr/cffr_{{ version }}.tar.gz
+  sha256: 9e890db13df829c28fabaa0674b84d4c32b0ce9e7c76e9b3665f019e9b1320f5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, rmarkdown, testthat (>= 3.0.0)
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-crayon >=1.4.1
+    - r-desc >=1.3.0
+    - r-jsonlite >=1.7.2
+    - r-jsonvalidate >=1.1.0
+    - r-yaml >=2.2.1
+
+  run:
+    - r-base
+    - r-crayon >=1.4.1
+    - r-desc >=1.3.0
+    - r-jsonlite >=1.7.2
+    - r-jsonvalidate >=1.1.0
+    - r-yaml >=2.2.1
+
+test:
+  commands:
+    - $R -e "library('cffr')"           # [not win]
+    - "\"%R%\" -e \"library('cffr')\""  # [win]
+
+about:
+  home: https://docs.ropensci.org/cffr/, https://github.com/ropensci/cffr
+  license: GPL-3.0-or-later
+  summary: The Citation File Format version 1.2.0 <doi:10.5281/zenodo.5171937> is a human and
+    machine readable file format which provides citation metadata for software. This
+    package provides core utilities to generate and validate this metadata.
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - raivivek
```

Please add conda-forge/r
```suggestion
    - conda-forge/r
    - raivivek
```